src/views/docs-view/docs-view.module.css

```diff
@@ -1,3 +1,33 @@
 .mdxContent {
 	margin-top: 48px;
 }
+
+.contentWithVersions {
+	position: relative;
+
+	& .versionSwitcherWrapper {
+		width: 100%;
+		margin-bottom: 16px;
+
+		@media (--dev-dot-tablet-up) {
+			margin-bottom: 0;
+			width: auto;
+			position: absolute;
+			right: 0;
+			top: 0;
+		}
+	}
+
+	& .mdxContent {
+		@media (--dev-dot-mobile) {
+			margin-top: 0;
+		}
+	}
+
+	@media (--dev-dot-tablet-up) {
+		& h1:first-child,
+		& :global(.landing-hero) {
+			margin-right: 25%;
+		}
```

Comment:
Nice! Can we add a comment above this block just explaining what it's for and why we're doing it?

Reply:
[Added here](https://github.com/hashicorp/dev-portal/pull/945/files#diff-4d5c3e4fe15d4318a0f3ae5ea5bdc0b6cfbf15494fd7ac1a26c63a1cac2fbbdfR32-R33) 😄